apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -120,13 +120,20 @@ export function ConversationView({
         data-qa="conversation-name"
       >
         <Tooltip
-          tooltip={t(
-            getEntityNameError(isNameInvalid, isInvalidPath, isExternal),
-          )}
-          hideTooltip={!isNameOrPathInvalid}
-          triggerClassName="block max-h-5 flex-1 truncate whitespace-pre break-all text-left"
+          tooltip={
+            conversation.name ||
+            t(getEntityNameError(isNameInvalid, isInvalidPath, isExternal))
+          }
+          triggerClassName=" max-h-5 flex-1 truncate whitespace-pre text-left"
+          contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
```

Comment:
Why did you change these classes?

Reply:
I removed the 'block' so that it would no longer be a block element, and added a maximum size for phones and larger devices, because the tooltip was stretched to fill the entire width of the screen.